### DISCUSSION TITLE
Generation or Record key

### DISCUSSION THREAD

**Discussion by lslavkov** [sorry-cypress/sorry-cypress#430] [Q&A] [answered]
Hi, I have questions regarding Record key.

Only place I could get it is on official Cypress Dashboard when creating project. Issue is that I want to generate my own Record key but there is no guide for that on official documentation. 

Is there any guideline for generating Record key?

**Comment by agoldis** [comment]
@lslavkov you don't need a record key for sorry-cypress. Just use any arbitrary value.

**Reply by lslavkov** [reply]
Can you elaborate on that?

**Comment by agoldis** [comment, marked as answer]
I do - sorry-cypress is self-hosted, so there is no need distinguish between different customers and enforce limits via the record key.

**Reply by lslavkov** [reply]
I see, thanks for answering!
